platform_plugin_superset/extensions/filters.py

```diff
@@ -0,0 +1,145 @@
+"""
+Open edX Filters needed for Superset integration.
+"""
+import os
+
+import pkg_resources
+from crum import get_current_user
+from django.conf import settings
+from django.template import Context, Template
+from openedx_filters import PipelineStep
+from supersetapiclient.client import SupersetClient
+from web_fragments.fragment import Fragment
+
+os.environ["OAUTHLIB_INSECURE_TRANSPORT"] = "1"
```

Comment:
why are we modifying this here? what does it affect?

Reply:
This is was extracted from here: https://github.com/eduNEXT/aspects-superset-aside/pull/2/files#diff-e5ef7509835b73d0294668644b6133c454ad335edf6a41742fe278b064a0236cR15-R18

At this point, I don't remember what it does

Reply:
what if we try it without it?

Reply:
Fixed in #3